app/views/thredded/messageboards/_messageboard.html.erb

```diff
@@ -19,6 +19,8 @@
                 user: messageboard.last_user.thredded_display_name %>
         </p>
       <% end %>
+      <br>
```

Comment:
Perhaps it will look better with the icon in the top right corner?
Will also save space on mobile.

Reply:
I'm not too sure about this. Moving it to far right would destroy the symmetry. Your space point is valid though. I too thought about it but I traded that off because this was the most pleasing for me. Perhaps we could use another icon or something else for providing visual information?

I tried this, didn't find it appealing either: 
![image](https://user-images.githubusercontent.com/19175526/32208030-e4bef7c4-be25-11e7-948e-8cd3da11fa10.png)

Reply:
How about moving it right after the title (on the same line)?

Reply:
I think it'd be out of symmetry too. That was the first thing that I tried because it was the easiest.

Reply:
OK, let's go with the one in the screenshot then!